pkg/auth/setup/setup.go

```diff
@@ -258,8 +226,12 @@ func CreateInitialAdminUserWithKeys(ctx context.Context, authService auth.Servic
 }
 
 func CreateBaseGroups(ctx context.Context, authService auth.Service, cfg *config.Config, ts time.Time) error {
+	// TODO (niro): need to remove that when transitioning to external auth ACLs server
+	if cfg.IsAuthUISimplified() && cfg.Auth.API.Endpoint != "" {
```

Comment:
I don't understand the 3 arms of logic here.  Can you sketch what the func will look like when the transition finishes?

Reply:
When the transition finishes there will be no more call to CreateACLBaseGroups.
This part is done on the ACL server side (whether as part of a first `run` or an explicit `setup` command is still TBD.
For RBAC it will still create the base groups via lakeFS.